### ISSUE TITLE
Helm chart `lakefs-1.3.3` is not available yet

### ISSUE THREAD

**Issue by 0xmeyer** [treeverse/charts#271]
I am currently unable to install the newly released Helm chart as it is not available yet.

```bash
$ helm repo update
...
$ helm search repo lakefs
NAME            CHART VERSION   APP VERSION     DESCRIPTION                                  
lakefs/lakefs   1.3.2           1.28.0          A Helm chart for running LakeFS on Kubernetes
```

Best regards :)

**Comment by itaiad200** [comment]
Fixed by #273 
@0xmeyer can you verify?

**Comment by 0xmeyer** [comment]
Works now, thank you!
